superset/tasks/schedules.py

```diff
@@ -538,16 +543,15 @@ def schedule_alert_query(  # pylint: disable=unused-argument
             logger.info("Ignoring deactivated alert")
             return
 
+        if schedule.sql_observers:
+            sql = schedule.sql_observers[0].sql
```

Comment:
could there be multiple observers?
if not, let's change sql_observers to sql_observer